themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html

```diff
@@ -20,7 +20,7 @@
 							</div>
 							<div class="col d-flex align-items-start justify-content-center justify-content-sm-end">
 								<button class="collapsed btn btn-close" type="button" data-toggle="collapse" data-target="#collapse-{{ $index }}" aria-expanded="true" aria-controls="collapse-{{ $index }}">
-									<img class="img-fluid" src="/img/close-icon.svg">
+									<img class="img-fluid" src="/img/close-icon.svg" alt="">
```

Comment:
Why are we using an empty alternative description? I would like to see a real description like "Closing Icon 'X'"